content/docs/user-guide/components/reference/vegetation/layer-spawner.md

```diff
@@ -1,31 +1,63 @@
-description: ' Use the Vegetation Layer Spawner component to define areas and rules
-  for procedurally placing dynamic vegetation or other static meshes in Open 3D Engine. '
 title: Vegetation Layer Spawner Component
+description: Use the Vegetation Layer Spawner component to define areas and rules for procedurally placing dynamic vegetation or other static meshes in Open 3D Engine (O3DE).
+weight: 500
 ---
 
-
-
-Use the **Vegetation Layer Spawner** component to define areas and rules for procedurally placing dynamic vegetation or other static meshes in your O3DE level.
-
-To use the **Vegetation Layer Spawner** component, you must enable the **Vegetation** gem.
-
-With the **Vegetation Layer Spawner** component, you can do the following:
-+ Create vegetation within a user-defined area at runtime
-+ Configure layer settings to determine the depth or relative ordering in which to apply vegetation layers
-+ (Optional) Add modifier and filter components to add variation to placed vegetation
-+ (Optional) Use selector components to determine which vegetation assets to place at a given location
-+ Control the preview settings for attached components
-+ Inherit behavior from a parent **Vegetation Layer Blenders** component
-
-Add the following required components when using the **Vegetation Layer Spawner** component:
-+ A **Shape** or **Vegetation Reference Shape** component to define the vegetation's spawn area
-+ The **Vegetation Asset List** component to list mesh assets, material assets, and other settings for the vegetation
-
-The following optional components change the **Vegetation Layer Spawner** component's behavior:
-+ **Vegetation Filter** components use various conditions to determine whether to create vegetation
-+ **Vegetation Modifier** components change the appearance of generated vegetation
-+ **Vegetation Selector** components determine which asset descriptors are selected for creation
-
-For information on how to use the **Vegetation Layer Spawner**, see [Dynamic vegetation](/docs/user-guide/gems/reference/environment/vegetation/).
+Use the **Vegetation Layer Spawner** component to define areas and rules for procedurally placing dynamic vegetation or other static meshes in your **Open 3D Engine (O3DE)** level.
+
+With the Vegetation Layer Spawner component, you can do the following:
+- Create vegetation within a user-defined area at runtime.
+- Configure layer settings to determine the depth or relative ordering in which to apply vegetation layers.
+- Add [Vegetation Modifier](./../vegetation-modifiers/) and [Vegetation Filter](./../vegetation-filters/) components to add variation to placed vegetation.
+- Use the [**Vegetation Asset Weight Selector**](vegetation-asset-weight-selector) component to determine which vegetation assets to place at a given location.
+- Control the preview settings for attached components.
+Inherit behavior from a parent [**Vegetation Layer Blender**](vegetation-layer-blender) component.
```

Comment:
Another instance of inconsistency with bolded links. Notice the component links in line 13 are not bolded, but the ones in line 14 and 16 are.